plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/compare/revisionsList/RevisionListViewImpl.java

```diff
@@ -0,0 +1,214 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.ext.git.client.compare.revisionsList;
+
+import com.google.gwt.cell.client.TextCell;
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.event.dom.client.ClickEvent;
+import com.google.gwt.event.dom.client.ClickHandler;
+import com.google.gwt.event.dom.client.DoubleClickEvent;
+import com.google.gwt.event.dom.client.DoubleClickHandler;
+import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.user.cellview.client.CellTable;
+import com.google.gwt.user.cellview.client.Column;
+import com.google.gwt.user.client.ui.Button;
+import com.google.gwt.user.client.ui.ScrollPanel;
+import com.google.gwt.user.client.ui.TextArea;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.gwt.view.client.SelectionChangeEvent;
+import com.google.gwt.view.client.SingleSelectionModel;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import org.eclipse.che.api.git.shared.Revision;
+import org.eclipse.che.ide.ext.git.client.DateTimeFormatter;
+import org.eclipse.che.ide.ext.git.client.GitLocalizationConstant;
+import org.eclipse.che.ide.ext.git.client.GitResources;
+import org.eclipse.che.ide.ui.window.Window;
+
+import javax.validation.constraints.NotNull;
+
+import java.util.List;
+
+/**
+ * The implementation of {@link RevisionListView}.
+ *
+ * @author Igor Vinokur
+ */
+@Singleton
+public class RevisionListViewImpl extends Window implements RevisionListView {
+    interface BranchViewImplUiBinder extends UiBinder<Widget, RevisionListViewImpl> {
+    }
+
+    private static BranchViewImplUiBinder ourUiBinder = GWT.create(BranchViewImplUiBinder.class);
```

Comment:
Please, name UI-binder in accordance to the view's name. If you have ```RevisionListViewImpl``` then you should have  ```RevisionListViewImplUiBinder``` but not ```BranchViewImplUiBinder```